x/src/test.rs

```diff
@@ -17,16 +23,66 @@ pub struct Args {
     #[structopt(long, short)]
     /// Only run unit tests
     unit: bool,
+
+    #[structopt(long)]
+    /// Do not fast fail the run if tests (or test executables) fail.
+    no_fail_fast: bool,
+    #[structopt(long)]
+    /// Do not run tests, only compile the test executables.
+    no_run: bool,
+
+    #[structopt(long, parse(from_os_str))]
+    /// Setting an hmtlcovdir will result in grcov being run for the workspace, or crate the
+    /// command is executed from.   A clean will be performed before running the tests.
+    htmlcovdir: Option<OsString>,
```

Comment:
To be uniform with the other options above this should be `html_cov_dir`. But I'd personally like a name like `cov_out_dir`

Reply:
I had earlier and plan to add in an lcov, option (generate lcov file, then html file from lcov -- it's different).   Took that out of this pr.   Will use html_cov_dir?  Naming this is hard since grcov -> html is one path and gcov -> lcov -> html is another.

Reply:
Ahh, I see. What about breaking this up into two options: the coverage engine and the output dir?

Reply:
The engine stays the same but unfortunately converting the gcda files' data to lcov data files loses data, which is reflected in any report generated from the lcov data.  Codecov is currently being feed lcov data.